addons/html_builder/static/src/plugins/button_option.js

```diff
@@ -0,0 +1,16 @@
+import { Plugin } from "@html_editor/plugin";
+import { registry } from "@web/core/registry";
+
+class ButtonOptionPlugin extends Plugin {
+    static id = "ButtonOption";
```

Comment:
The convetion is camelcase without uncapitalized `buttonOption`.
Otherwise it would be `dependencies.ButtonOption`

Reply:
Thanks for checking :smile_cat: Just a question, cause I checked the other options in the current codebase, it seems the options have their ID first letter capitalized. But for the plugins in core, the ID is camelcase

Reply:
Yes we need to do a big clean it think ^^